frontend/src/metabase/core/components/TabButton/TabButton.styled.tsx

```diff
@@ -18,6 +18,7 @@ export const TabButtonInputWrapper = styled.span`
 export const TabButtonInputResizer = styled.span`
   visibility: hidden;
   white-space: pre;
+  padding-right: 2px;
```

Comment:
A small amount of padding is needed to prevent the border on the input from cutting off the text.